[2023-02-04 10:00:00,544] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T09:55:00+00:00 [queued]>
[2023-02-04 10:00:00,552] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T09:55:00+00:00 [queued]>
[2023-02-04 10:00:00,553] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 10:00:00,553] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-04 10:00:00,554] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-04 10:00:00,567] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): bash_task> on 2023-02-04 09:55:00+00:00
[2023-02-04 10:00:00,572] {standard_task_runner.py:52} INFO - Started process 13335 to run task
[2023-02-04 10:00:00,575] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-new-dag', 'bash_task', 'scheduled__2023-02-04T09:55:00+00:00', '--job-id', '317', '--raw', '--subdir', 'DAGS_FOLDER/new_dag.py', '--cfg-path', '/tmp/tmp34x59ld8', '--error-file', '/tmp/tmpw0e6avig']
[2023-02-04 10:00:00,575] {standard_task_runner.py:80} INFO - Job 317: Subtask bash_task
[2023-02-04 10:00:00,638] {task_command.py:371} INFO - Running <TaskInstance: yandex-new-dag.bash_task scheduled__2023-02-04T09:55:00+00:00 [running]> on host 39df186e1726
[2023-02-04 10:00:00,714] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-new-dag
AIRFLOW_CTX_TASK_ID=bash_task
AIRFLOW_CTX_EXECUTION_DATE=2023-02-04T09:55:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-04T09:55:00+00:00
[2023-02-04 10:00:00,715] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-02-04 10:00:00,716] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'Hello, World!'"]
[2023-02-04 10:00:00,726] {subprocess.py:85} INFO - Output:
[2023-02-04 10:00:00,727] {subprocess.py:92} INFO - Hello, World!
[2023-02-04 10:00:00,728] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-04 10:00:00,760] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-new-dag, task_id=bash_task, execution_date=20230204T095500, start_date=20230204T100000, end_date=20230204T100000
[2023-02-04 10:00:00,787] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-04 10:00:00,824] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
